source/mongodump.txt

```diff
@@ -279,16 +281,18 @@ Data Exclusion
 :binary:`~bin.mongodump` excludes the content of the ``local`` database
 in its output.
 
-:binary:`~bin.mongodump` output only captures the documents in the
+:binary:`~bin.mongodump` output only exports the documents in the
 database and does not include index data. :binary:`~bin.mongorestore`
 or :binary:`~bin.mongod` must then rebuild the indexes after restoring
 data.
 
-If using :manual:`read-only views </core/views>`,
-:binary:`~bin.mongodump` only captures a view's metadata: it
-does not create a binary export of the documents included in the view.
-To capture the documents in a view use
-:option:`--viewsAsCollections <mongodump --viewsAsCollections>`.
+If you're using :manual:`views </core/views>`:
+
+- By default, :binary:`~bin.mongodump` only exports a view's metadata.
```

Comment:
Farther down there's a note that only views are exported. Consider mentioning that here too.

Reply:
Done.